.github/workflows/helm-release.yaml

```diff
@@ -72,29 +77,68 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
 
-      - name: Set up python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "${{ inputs.python-version }}"
-
       - name: Bump version
         id: release-tag
-        uses: bakdata/ci-templates/actions/bump-version@v1.0.0
+        uses: bakdata/ci-templates/actions/bump-version@v1.5.2
         with:
           release-type: ${{ inputs.release-type }}
+          working-directory: ${{ inputs.charts-dir }}
+
+      - name: Download previous Helm index & packages
+        run: curl -O ${{ inputs.page-url }}/index.yaml
+        shell: bash
+        working-directory: ${{ inputs.charts-dir }}
+
+      - name: Helm package
+        uses: bakdata/ci-templates/actions/helm-package@feat/helm-release
+        with:
+          charts-dir: ${{ inputs.charts-dir }}
 
-      - name: Commit and push bumpversion.cfg file
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+      - name: Commit and push Chart.yaml & .bumpversion.cfg file
+        uses: bakdata/ci-templates/actions/commit-and-push@feat/helm-release
         with:
           ref: ${{ inputs.ref }}
           commit-message: "Bump version ${{ steps.release-tag.outputs.old-tag }} → ${{ steps.release-tag.outputs.release-tag }}"
           github-username: ${{ secrets.github-username }}
           github-email: ${{ secrets.github-email }}
           github-token: ${{ secrets.github-token }}
 
-      - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.2.1
+      - name: Find previous package versions
+        id: packages
+        uses: mikefarah/yq@master
+        with:
+          cmd: yq '.entries.[] | map(.urls.[]) | join(",")' ${{ inputs.charts-dir }}/index.yaml
+
+      - name: Prepare artifact
+        run: |
+          mkdir artifact
```

Comment:
Also not a blocker, but it might be useful to make this an input parameter, so that users can configure it in case there is already a directory called artifact